### PR TITLE
Added Salty Bet (www.saltybet.com)

### PR DESCRIPTION
**Pull request by magumboi**
I hope you like it!

### CODE REVIEW

File: websites/S/Salty Bet/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Rare Akuma ☆",
+    "id": "250044591686811669"
+  },
+  "url": "www.saltybet.com",
+  "description": {
+    "en": "SaltyBet (also known as Salty's Dream Cast Casino) is a betting site where users may bet fake money on the outcomes of A.I.-controlled M.U.G.E.N battles. The fights are broadcasted through Twitch 24/7, and feature a variety of characters and stages. SaltyBet is one of the most active M.U.G.E.N-related websites, with over 22 million views on its Twitch stream."
+  },
```
Comment (Bas950) [inline websites/S/Salty Bet/dist/metadata.json:10]: Make the description shorter

File: websites/S/Salty Bet/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "service": "Salty Bet",
+  "version": "1.1.3",
```
Comment (Bas950) [inline websites/S/Salty Bet/dist/metadata.json:12]: Version 1.0.0 on release

File: websites/S/Salty Bet/presence.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
+const presence = new Presence({
+  clientId: "802246778010730548" //The client ID of the Application created at https://discordapp.com/developers/applications
+}),
```
Comment (Bas950) [inline websites/S/Salty Bet/presence.ts:3]: remove comment

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by magumboi** [comment]
done 👌

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**